charter.md

```diff
@@ -0,0 +1,51 @@
+# GitOps Working Group Charter
+
+## Introduction
+
+This document outlines the charter for the GitOps Working Group, a [time-limited Working Group](https://github.com/cncf/toc/blob/main/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs) under the [CNCF App Delivery SIG](https://github.com/cncf/sig-app-delivery).
+
+## Mission Statement
+
+The focus of the GitOps WG is to clearly define a vendor-neutral, principle-led meaning of GitOps.
+This will establish a foundation for interoperability between tools, conformance, and certification. Lasting programs, documents, and code are planned to live within the [OpenGitOps](https://github.com/open-gitops) project.
+
+## Goals
+
+- Open governance
+- Project/vendor neutral
+- Create education materials and promote awareness and understanding of GitOps terminology and approaches
```

Comment:
Would change wording to:

- Promote awareness and understanding of GitOps terminology and approaches by creating education materials, organizing events and advocating throughout the community

Reply:
Added with commit co-authorship 👍